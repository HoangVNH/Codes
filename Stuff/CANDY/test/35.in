10
24
VVXXXXXXXXDDDDDDDDXXXXXX
49
XXXXXXXXXDVVXXDDDDDDDDXXXDDDDDDDDDDDDDDXDDDVVVVVV
41
DDDVVVVDDDDDXXXVVDDDDDDDVVVVDVXXXXXXXDDDD
25
XXDVVVVVXXXXXDXXXXXVVVVDD
33
XXXXVVVVVXXXDDDDVVVVVVVDDXDDDDVXX
40
VXXXXXDDDDDDDDXXXVVDDDDVXXXXXVVVVVVVDDDD
20
VVVVVVVVVVVVDDDVVVVV
38
XXXXXXXXXXVVVXXXXXDDXXXDVVXXXXXXXXXXXX
34
XXXVVVXXXXVVVVDDDDDDDDDVVVVXXXVVVV
40
VVVVVXVVVVVVVVVVVDDDDXXXVVXXXVVVVVVVVVVV
