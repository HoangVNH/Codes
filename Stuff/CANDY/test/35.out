22
48
40
24
32
39
19
37
33
39
